### PR TITLE
Add drewkeys/iskar default layout

### PR DESCRIPTION
**Pull request by drewpyun**
## Description
Added the default layout to iskar. added the iskar_default.json to keymaps folder (under "i").
Initial PR for QMK files are here -> https://github.com/qmk/qmk_firmware/pull/16562

### CODE REVIEW

File: public/keymaps/i/iskar_default.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "keymap": "default",
+  "commit": "403536986d4a4808a02bb0e0a2d0de8ab4551541",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+      "QK_GESC",  "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_6", "KC_7", "KC_8",    "KC_9",   "KC_0",     "KC_MINS", "KC_EQL",  "KC_DEL",  "KC_BSPC", "KC_HOME",
+      "KC_TAB",   "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I",    "KC_O",             "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS", "KC_INS",
+      "KC_CAPS",  "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K",    "KC_L",             "KC_SCLN", "KC_QUOT", "KC_ENT",           "KC_PGUP",
+      "KC_LSFT",  "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "KC_SLSH",                    "KC_RSFT", "KC_UP",   "KC_PGDN",
+      "KC_LCTRL", "KC_LGUI", "KC_LALT",        "KC_SPC",                    "MO(1)", "KC_RCTRL",                   "KC_LEFT", "KC_DOWN", "KC_RGHT"
```
Comment (fauxpark) [inline public/keymaps/i/iskar_default.json:12]:
```suggestion
      "KC_LCTL", "KC_LGUI", "KC_LALT",        "KC_SPC",                    "MO(1)", "KC_RCTL",                   "KC_LEFT", "KC_DOWN", "KC_RGHT"
```

File: public/keymaps/i/iskar_default.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    ],
+    [
+      "KC_GRAVE", "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_F6", "KC_F7", "KC_F8",  "KC_F9",   "KC_F10",     "KC_F11",  "KC_F12",   "KC_TRNS",   "KC_TRNS",  "KC_TRNS",
```
Comment (fauxpark) [inline public/keymaps/i/iskar_default.json:15]:
```suggestion
      "KC_GRV", "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_F6", "KC_F7", "KC_F8",  "KC_F9",   "KC_F10",     "KC_F11",  "KC_F12",   "KC_TRNS",   "KC_TRNS",  "KC_TRNS",
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by noroadsleft — CHANGES_REQUESTED** [review]
The file contents are fine, but this file needs to be moved/renamed to work as intended:

```
git mv public/keymaps/i/iskar_default.json public/keymaps/d/drewkeys_iskar_default.json
git commit -m "rename to public/keymaps/d/drewkeys_iskar_default.json"
```

File: public/keymaps/d/drewkeys_iskar_default.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "keymap": "default",
+  "commit": "403536986d4a4808a02bb0e0a2d0de8ab4551541",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+      "QK_GESC",  "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_6", "KC_7", "KC_8",    "KC_9",   "KC_0",     "KC_MINS", "KC_EQL",  "KC_DEL",  "KC_BSPC", "KC_HOME",
+      "KC_TAB",   "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I",    "KC_O",             "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS", "KC_INS",
+      "KC_CAPS",  "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K",    "KC_L",             "KC_SCLN", "KC_QUOT", "KC_ENT",           "KC_PGUP",
+      "KC_LSFT",  "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "KC_SLSH",                    "KC_RSFT", "KC_UP",   "KC_PGDN",
+      "KC_LCTRL", "KC_LGUI", "KC_LALT",        "KC_SPC",                    "MO(1)", "KC_RCTRL",                   "KC_LEFT", "KC_DOWN", "KC_RGHT"
```
Comment (noroadsleft) [inline public/keymaps/d/drewkeys_iskar_default.json:12]:
```suggestion
      "KC_LCTL",  "KC_LGUI", "KC_LALT",        "KC_SPC",                    "MO(1)", "KC_RCTL",                    "KC_LEFT", "KC_DOWN", "KC_RGHT"
```

File: public/keymaps/d/drewkeys_iskar_default.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    ],
+    [
+      "KC_GRAVE", "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_F6", "KC_F7", "KC_F8",  "KC_F9",   "KC_F10",     "KC_F11",  "KC_F12",   "KC_TRNS",   "KC_TRNS",  "KC_TRNS",
```
Comment (noroadsleft) [inline public/keymaps/d/drewkeys_iskar_default.json:15]:
```suggestion
      "KC_GRV",   "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_F6", "KC_F7", "KC_F8",  "KC_F9",   "KC_F10",     "KC_F11",  "KC_F12",   "KC_TRNS",   "KC_TRNS",  "KC_TRNS",
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Pull request merged**